config/default.js

```diff
@@ -376,13 +376,5 @@ module.exports = {
   // See: https://github.com/mozilla/addons-frontend/issues/8680
   enableFeatureRemoveSearchTools: false,
 
-  // The withExperiment HOC relies on this config to enable/disable A/B
-  // experiments.
-  experiments: {
```

Comment:
Would it help to define an empty object here with a comment advising the reader to consider enabling experiments in `default-amo` or `default-disco` instead?

Reply:
I thought about doing that, but considered it unnecessary. But I can add that back in.

Reply:
Actually I was concerned that even defining an empty object here might accidentally encourage someone to add an experiment here. Adding a comment should ideally prevent that, but not even defining an `experiments` property here seems like an even better way of preventing people from thinking this would be a good place to add an experiment.

WDYT?

Reply:
Hmm, I see tests are failing, so we might need one here anyway.

Reply:
we always have to define a default config because there are configs for tests and uitests.